src/lib/aws/construct/static-asset-deployment/main.ts

```diff
@@ -49,14 +59,79 @@ export class StaticAssetDeployment extends CommonConstruct {
     this.staticAssetBucket = this.s3Manager.createS3Bucket(`${this.id}-sa-bucket`, this, this.props.staticAssetBucket)
   }
 
+  /**
+   * @summary Loads the static asset bucket
+   */
+  protected loadAssetBucket() {
```

Comment:
`resolveAssetBucket`

Reply:
this uses existing `createBucket` method on `s3Manager` as per conversation